docs/_templates/page.html

```diff
@@ -0,0 +1,227 @@
+{% extends "base.html" %}
```

Comment:
Should this be in https://github.com/flyteorg/furo/blob/main/src/furo/theme/furo/page.html or here?

Reply:
we're no longer using that fork of furo in the monodocs